SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-private-mdc-mapping.adoc

```diff
@@ -0,0 +1,66 @@
+[#ref_private_coding_system_note]
+==== Private MDC Codes Consideration
+The default coding system utilized in SDC is *"MDC"* (Medical Device Communications) also known as *"ISO/IEEE 11073-10101"* nomenclature.
+
+In addition to the standard codes, *MDC* defines private code ranges that can be used by medical device vendors for defining their own codes, for example, for concepts for which standard codes do not exist. Private codes should be avoided whenever possible since this undermines interoperability.
+
+Coded elements in SDC using *MDC* private codes shall also contain a *pm:Translation* element that defines the vendor-specific coding system.
+
+The following example uses the SDC *pm:Type* to demonstrate the mapping of private MDC codes.
+
+[#ref_tbl_private_code_mapping]
+.Private Code Mapping (CWE data type example)
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|CWE-1
+|Identifier
+|pm:Type
+/@Code
+| This attribute contains the private *MDC* code.
+
+|CWE-2
+|Text
+|"*MDC_PRIVATE_*<CWE-1>"
+|Since this is a private code, a MDC RefId does not exist for the code. The field shall be set to a private RefId which consists of the prefix *"MDC_PRIVATE_"* plus the private code set in CWE-1.

```

Comment:
This is an interesting one. Aren't private codes supposed to be written as `MDCX_` or is this something different? I like the idea of marking private codes this way but didn't know that is a rule.

Reply:
@d-gregorczyk: this is what we have decided at the SDPi workshop in Luebeck. My understanding is that MDCX_ is for pre-liminary codes which are not in the standard right now - but I could be wrong.

Reply:
Comment from Todd in the SDPi meeting on 12/2/2022: Todd confirmed that MDCX_ is used for proposed terms and should not used for the private codes. So, we leave it as is.